frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -123,19 +123,26 @@ export const infiniteListLogic = kea<infiniteListLogicType>({
                         await breakpoint(1)
                     }
 
-                    const { isExpanded, remoteEndpoint, scopedRemoteEndpoint, searchQuery } = values
+                    const { isExpanded, remoteEndpoint, scopedRemoteEndpoint, searchQuery, excludedProperties } = values
 
                     if (!remoteEndpoint) {
                         // should not have been here in the first place!
                         return createEmptyListStorage(searchQuery)
                     }
 
-                    const searchParams = {
+                    let searchParams = {
                         [`${values.group?.searchAlias || 'search'}`]: searchQuery,
                         limit,
                         offset,
                     }
 
+                    if (excludedProperties) {
+                        searchParams = {
+                            ...searchParams,
+                            excluded_properties: JSON.stringify(excludedProperties),
+                        }
+                    }
```

Comment:
It could as well be a `const` here, and you can do just `searchParams.excluded_properties = JSON.stringify(excludedProperties)`. Until we put variables into kea/redux, you can mutate them in whatever way you want to.

Reply:
Done!